must/must.go

```diff
@@ -0,0 +1,41 @@
+package must
+
+import "strings"
+
+func Must[T any](obj T, err error) T {
+	if err != nil {
+		panic(err)
+	}
+
+	return obj
+}
+
+func MustBeOk[T any](value T, ok bool, args ...error) T {
+	if !ok {
+		panic(BuildErrorMessage("must be ok", args))
+	}
+	return value
+}
+
+func MustNotBeOk[T any](value T, ok bool, args ...error) T {
+	if ok {
+		panic(BuildErrorMessage("must not be ok", args))
+	}
+	return value
+}
```

Comment:
1. Both funcs do very similar things, except for the test condition (!ok vs ok). U can eliminate duplication by creating a common internal function.

Smth like:

```golang
func assertCondition[T any](value T, condition bool, defaultMsg string, args ...error) T {
	if !condition {
		panic(BuildErrorMessage(defaultMsg, args...))
	}
	
	return value
}
```